sources/packages/backend/apps/db-migrations/src/sql/Student/Rollback-update-gender-options.sql

```diff
@@ -0,0 +1,27 @@
+UPDATE
+  sims.students
+SET
+  gender = 'male'
+WHERE
+  gender = 'man';
+
+UPDATE
+  sims.students
+SET
+  gender = 'X'
+WHERE
+  gender = 'nonBinary';
+
+UPDATE
+  sims.students
+SET
+  gender = 'female'
+WHERE
+  gender = 'woman';
+
+UPDATE
+  sims.students
+SET
+  gender = preferNotToAnswer
+WHERE
+  gender IS NULL;
```

Comment:
`gender` is a `not null` column hence it will never be null. Am I missing something?

Reply:
updated